config/config.exs

```diff
@@ -31,6 +31,27 @@ config :animina, :ai_message_help_price, 20
 # Configures whether to have ai automatically respond to messages
 config :animina, :autoreply_messages_with_ai, false
 
+# Configures default maximum age for potential partners
+config :animina, :default_potential_partner_maximum_age, 100
+
+# Configures default minimum age for potential partners
+config :animina, :default_potential_partner_minimum_age, 18
+
+# confugures the difference in age if no maximum or minimum age is set
+config :animina, :default_potential_partner_age_difference, 10
```

Comment:
What does this mean? If I am 50 years old will that mean that the range is from 40 - 60 or is it from 45 - 55? Either improve the actual code or add a better comment to it.

Reply:
It means range is from 40 and 60 , let me use betteeer words

Reply:
Let's call it `default_partner_height_offset`

Not perfect but good for now.